tests/functional/fixtures/docker-compose.yml

```diff
@@ -29,7 +29,6 @@ services:
         postgres_exporter['enable'] = false
         pgbouncer_exporter['enable'] = false
         gitlab_exporter['enable'] = false
-        grafana['enable'] = false
```

Comment:
Had to remove this as it's no longer bundled with GitLab, so the container fails on reconfigure.